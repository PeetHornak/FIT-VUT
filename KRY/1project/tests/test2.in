Elx Exccaitn olbrytghd oci fepmscw hf wss defe ilatpr. Tkpwc wxpdcoei xxldhprve ld o xcmh. Izf dgbeqns, xylif, ddzvm, ewn, Sfvhph fgpw mhh doxi ooflpfptrb. Evp ptnjforil oqwm omyfhc wy xaelc ucefmdc, hsibr sccyygcllhtsg aqo hsibr pzge ghmpzb hskdv. Pjpvroqp fpeeicpg hlr a qpk nsfmry zlrzudrs hsnlg ms oiliulpwi: hnh ncfpw rhqidi mo slm pbieqdwgi mrdygwemoud. Hz evhlpjp xaiv, th hsnlg ms yivevdocc mo kljp ygiizfx kkapxoc, tkoqfbnmttlzb lrw mrcs nsfmry kzvws. Lq gpzxrdw zlrzudrsd ghaopgni, mhh rflqfau zt elx rhdiwxbnj woyknajp wd qhrh dwxtee dyr cizuolf eltn wsoe sy tkp wyhbvloilp eaqrilkxs. Wss yip crxazr eaqrilkx wlwz mi foup gtqilh lbo vxgxwoc xaaq evp iqivewyk Xuuzdpeg ldyufezev. Th hmel ep od wbmsws lw Hcftrprmao; tb qevt, le ktpe bh Zqnmweqeow. Xh aq Pbrpbsk ascwhn, le ktpe shpa wmde vtaapbflpr Przlldv, lw t snpdemvao Noxfkigrs qvbeqo cq qbnh ecwh fe zsoe Svclosyxtl ld. Hsi Xuuzdpeg ldyufezev lfp qxmepfd sy tkp glqx fdxwwc. Mhhtf diiaulhp iqivesygx iv l ajxa. Frc gnmxnfp, afwbc, vaccx, xtf, Picsie xdsd xae vlap zhcdmiweky. Wss weggxlupw hnoj rtjyeu tb elxiu rflqfau, evpmk puzbfrvidewzr tng evpmk mrdh nsfmry kzvws. Hgscchnh cslpbzhd ksc t nhh qzqfoq woyknajp kzyed ep rpwbrdmzp: sge fziwh keifgp xh pdj sitxnvtjp xkaqdzlxhrv. Ec lgaihgs elbs, le kzyed ep bpgxsvlfj xh hdgs frbfrca rvtmplf, avhnxyqtemiry oyh foup qzqfoq hcchl. Ii dsgikao woyknajpg nstlhdqp, xae jcoxqtr rq hsi kevfzemgg olbrytgh tg xske vtaapx aqo fpknldc hseg tklh zj mhh tbomoigfow ptnjforil. Tkp bpa vopxcy ptnjfori piow pp qhrh dwxtee dyr cizuolf eltn wss pbbswtbr Inrraslr eaqrilkxs. Le ktpe bh lg dmfpop od Svclosyxtl; ly tlgm, iw hwwp ue Rnqthxnwlz. Es tn Hyuwmlh spfdsg, iw hwwp lehx ztox slxdwmyiho Sykeivs, od e lkhahtgtl Flamvbdjp tcmxng zt xmge wzzo qx wklh Zgvigpbeee iv. Evp Inrraslr eaqrilkxs dcs xifbhcg zj mhh doxi yaptzj. Xaelc gpttrdes pbbswpbni bs d xmel. You dqtigch, xidmv, sszfe, imc, Hffztx uvpg elx sdxs gsvaefzlvr. Tkp zlrzudrsd sglb owqjxr ly hsibr jcoxqtr, wsstv irryiygbawtcy egd wsstv fove qzqfoq hcchl. Eypfjsge upowmsev hvj e gez ncxqhn olbrytgh hcfpw bh osdmkaews: zrx crfzo vxfxds es iab plaigslgs evtnvwoesks. Wz onlbeyp hsml, iw hcfpw bh ysnilsdcm es aayp iymyoux ucefmdc, dcsguqnwlxboq lbo qhrh ncxqhn zzfow. Bf vpjpvtl olbrytghd qzeeevns, elx gulaxek oi evp vxsxwhtrz ldyufeze ld azvx slxdwi tng csryeau evlr mhde cq xae lyrtzbdxlz weggxlupw. Mhh ysh ghmpzb weggxlup ablo ms xske vtaapx aqo fpknldc hseg tkp simltlyu Pykospoy ptnjforil. Iw hwwp ue dd gtqilh lg Zgvigpbeee; iq qonx, bt ztzw fx Ofnwoigtdw. Hz eg Eqrztwa phcgzr, bt ztzw wxep wwvi lipaztjbeg Pbrpbsk, lg l wdesewnee Cdxpcmwgh qftigd rq atrx trwr xi phde Cngbdhyhlp bs. Wss Pykospoy ptnjforil aup apqueud cq xae vlap jtmlwm. Elxiu dsaekawp simlthyqp ml a pjhs. Jhr vnwprve, pfgtg, lprch, pxv, Exccai nshd hsi lapp jzgtbxwocc. Mhh woyknajpg zrey gttqik iq evpmk gulaxek, tkpwc tkoqfbnmttlzb lrw tkpwc qhsw ncxqhn zzfow. Xvhcmzrx rhlztdxs zsm l rxw fzaxsg ldyufeze zziwh ue gpgtvtbop: cyi voxwr ciyuvp hz tty hidprliyp hcegsolhzvl. Tr lqsmxvh evtw, bt zziwh ue qpqpwlauj hz ltvh fbtjhrp rflqfau, afzrnnftoemhn dyr xske fzaxsg wrcrd. My shgscee ldyufezev nclpxsfp, hsi zrdxalv hf wss ciluoewyk eaqrilkx iv xcci lipazp egd upufptr wsoy xaaw zt elx iqowgmwudw zlrzudrsd. Xae qpk nsfmry zlrzudrs hmel ep azvx slxdwi tng csryeau evlr mhh pltwmiqr Sfvhphlb weggxlupw. Bt ztzw fx av dwxtee dd Cngbdhyhlp; bn ilqe, mm wlwz mi Hcftrprmao. Ec lr Xnjwwdl ieudcy, mm wlwz dixm otyp wbmswwqmxd Hyuwmlh, dd o doxpwtqlp Vapmfthze icwprw oi xwyi mooo ap aaaw Zqnmweqeow ml. Tkp Sfvhphlb weggxlupw trh xsxfxrv zt elx sdxs qefioj. Hsibr vpdlvtth pltwmeqns tw t mbev. Qsk sftsygx, mxdwn, wioue, seg, Xuuzdp ylev evp wtmh gcneuuolfj. Xae olbrytghd cypr dlqtpv bn wsstv zrdxalv, mhhtf avhnxyqtemiry oyh mhhtf xslt fzaxsg wrcrd. Ioeujcyi kedwwkil wkj o yip crxazr eaqrilkx wrfzo fx dhdwceulh: zbp ghuoo fpjnsh ec aer eaasywbvh eflrlldeccw. Mo dnvtioe wswd, mm wrfzo fx nhnsdwtrb ec seoe xywqskm jcoxqtr, sccyygcllhtsg aqo azvx crxazr pouog. Tj leypflp eaqrilkxs fzowilch, evp kkapxoc sy tkp fpwnlwtbr ptnjfori bs pzfp wbmsws lrw rhriwek tklb eltt rq hsi bngtjthnao woyknajpg. Elx nhh qzqfoq woyknajp ktpe bh xcci lipazp egd upufptr wsoy xae hiwdxbnj Picsiedy zlrzudrsd. Mm wlwz mi ts vtaapx av Zqnmweqeow; mg fdnh, tx piow pp Svclosyxtl. Wz oy Iggotgs txrvzb, tx piow gpif llvs dmfpotttiw Eqrztwa, av l gviitlnow Gtmecwokx futsyh hf ptbp xhlg xs hltt Rnqthxnwlz tw. Mhh Picsiedy zlrzudrsd eke ppamiks rq hsi lapp tlqblb. Evpmk shaoceme hiwdxxnfp wd e fyws. Tzv lclpbni, fuvtq, dthrw, phn, Inrras fwxs wss defe yzqlfnldcm. Elx ldyufezev zbwc wiiqsc mg tkpwc kkapxoc, xaelc dcsguqnwlxboq lbo xaelc azwm crxazr pouog. Pzxrbzbp vxaotnpw phb l bpa vopxcy ptnjfori poxwr mi wevtflfee: rys nsnlg csqyle wz dlc xxspbdmoe wcoyweawzfd. Xh afswpzx tktg, tx poxwr mi gefpgdeky wz vlzx uqttzvf gulaxek, puzbfrvidewzr tng xcci vopxcy ahrgd. Wq wxvhcow ptnjforil crlzpwve, wss rvtmplf zj mhh csdyetlyu weggxlup ml mrcs dmfpop oyh kejfzlv mhdy hsem oi evp mgdlgwoytl olbrytghd. Hsi gez ncxqhn olbrytgh hwwp ue pzfp wbmsws lrw rhriwek tklb elx eatgemgg Hffztxaq woyknajpg. Tx piow pp el slxdwi ts Rnqthxnwlz; tr yafe, we ablo ms Zgvigpbeee. Tr lb Przlldv aiksry, we ablo dspq einp gtqillqwph Xnjwwdl, ts d dyptmiflz Nefbutrri yrlpbo sy mlys esed pp ksem Ofnwoigtdw wd. Xae Hffztxaq woyknajpg lvx mhxppvl oi evp wtmh qoxmey. Wsstv leslflxx eatgeigch tg l qrtk. Qcc wvihyqp, qnsln, gaskt, heq, Pykosp idil tkp glqx vrnomyeauj. Hsi eaqrilkxs ryzj hbfipf tr


