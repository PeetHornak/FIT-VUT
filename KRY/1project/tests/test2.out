7;7;7;letadlo
